node_lambnik/src/demo/datatype-demo.html

```diff
@@ -16,14 +16,14 @@ <h3>Datatype Demo</h3>
     <div id="map" style="width: 800px; height: 800px"></div>
     <div>
         <form id="layer-selector">
-            <input type="radio" name="layer" id="pwd_inlets" value="pwd_inlets">
-            <label for="pwd_inlets">pwd_inlets (POINT/EPSG:4326)</label>
+            <input type="radio" name="layer" id="inlets" value="inlets">
```

Comment:
Is there a reason for the "pwd_inlets"->"inlets" rename?  It's fine either way, just curious.

Reply:
That's the name of the file on OpenDataPhilly, since we're downloading all of the data at update time now I thought it might be a good idea to keep the file names the same across sources. Do you think it'd be unnecessarily confusing?

Reply:
No, it just seemed like an intentional shift and I wasn't sure what the intention was, but it's fine either way.

Reply:
Oh, now that I look, I think the other aspect of the change was that downloading the data straight from OpenDataPhilly leads to `inlets` being the name of the table in the database, and I thought it would be easier to update references than to remember to modify the table info.